core/glrender/src/main/java/edu/cmu/cs/dennisc/render/gl/imp/adapters/GlrCylinder.java

```diff
@@ -135,26 +186,31 @@ protected void pickGeometry(PickContext pc, boolean isSubElementRequired) {
       name = -1;
     }
     pc.gl.glPushName(name);
-    glCylinder(pc);
+    glCylinder(pc, false);
```

Comment:
also, the second argument should be true?

Reply:
This in `pickGeometry` that is used to determine interactions with the scene, so the full rendering is not required. In `renderGeometry` the isTextureEnabled does get checked and used (and is probably no longer needed above on line 125)

Reply:
Again, pickGeometry not renderGeometry, but this time I did remove the isTextureEnabled check in renderGeometry.